### ISSUE TITLE
wrong size chosen with luahbtex, harf mode

### ISSUE THREAD

**Issue by schlcht** [latex3/luaotfload#147]
Loading a font in `harf` mode will select a wrong size of it (or at least a different one than `base` and `node` modes). This only happens when running `luahbtex`, not `luatex` (TL '20). 

```
\input luaotfload.sty
\def\z{[lmroman12-regular.otf]:script=latn;+trep;+tlig;mode}
\font\x=\z=base;
\x Abc

\font\x=\z=node;
\x Abc

\font\x=\z=harf;
\x Abc
\bye
```

**Comment by zauguin** [comment]
This is fixed in the latest code in `dev`.

Actually `harf` still uses a different size than `base` and `node`, but the difference should not normally not be visible and the `harf` version is compatible with XeTeX (and IMO more correct). This is because `base` and `node` interpret the design size OpenType field as a number in TeX points, while OpenType fields normally use PostScript points.

**Comment by zauguin** [comment]
Fixed in v3.13.
